docs/ref/csrf.txt

```diff
@@ -92,6 +92,13 @@ protection for your views as outlined above.
     The CSRF token cookie is named ``csrftoken`` by default, but you can control
     the cookie name via the :setting:`CSRF_COOKIE_NAME` setting.
 
+    The CSRF header name is ``HTTP_X_CSRFTOKEN`` by default, but you can control
+    the header name via the :setting:`CSRF_HEADER_NAME` setting.
+
+    This header like any other http headers, before it appears in ``request.META``
```

Comment:
I got confused as well and my comment is incorrect, but I think this doc could still be improved (and maybe moved to the setting doc instead of saying "this can be whatever you want" there). Something like:

As with other HTTP headers in `request.META`, the header name received from the server is normalized by converting all characters to uppercase, replacing any hyphens with underscores and adding an 'HTTP_' prefix to the name. For example, if your client sends a `X-XSRF-TOKEN` header, the setting should be 'HTTP_X_XSRF_TOKEN`.